__test__/logic.test.js

```diff
@@ -0,0 +1,124 @@
+const logic = require('../src/js/pre-logic.js');
+let fakeState = {
+  collision: false,
+  board: {
+    size: 144,
+    rows: 12,
+    cellSize: 4,
+    walls: [
+      {x: 0, y: 0},
+      {x: 1, y: 0},
+      {x: 2, y: 0},
+      {x: 3, y: 0},
+      {x: 8, y: 0},
+      {x: 9, y: 0},
+      {x: 10, y: 0},
+      {x: 11, y: 0},
+      {y: 0, x: 0},
+      {y: 1, x: 0},
+      {y: 2, x: 0},
+      {y: 3, x: 0},
+      {y: 8, x: 0},
+      {y: 9, x: 0},
+      {y: 10, x: 0},
+      {y: 11, x: 0},
+      {x: 0, y: 11},
+      {x: 1, y: 11},
+      {x: 2, y: 11},
+      {x: 3, y: 11},
+      {x: 8, y: 11},
+      {x: 9, y: 11},
+      {x: 10, y: 11},
+      {x: 11, y: 11},
+      {y: 0, x: 11},
+      {y: 1, x: 11},
+      {y: 2, x: 11},
+      {y: 3, x: 11},
+      {y: 8, x: 11},
+      {y: 9, x: 11},
+      {y: 10, x: 11},
+      {y: 11, x: 11}
```

Comment:
In general, you should initialise only the state you use. In your case it makes sense because you have `pickRnd`.